Formula/libsoup@2.rb

```diff
@@ -0,0 +1,67 @@
+class LibsoupAT2 < Formula
+  desc "HTTP client/server library for GNOME"
+  homepage "https://wiki.gnome.org/Projects/libsoup"
+  url "https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.1.tar.xz"
+  sha256 "dc27a3b8fa30bc8ff950b5a75958757d2242e3e51e4e2d9c4e623ef75f793bf8"
+  license "LGPL-2.0-or-later"
+
+  keg_only :versioned_formula
+
+  deprecate! date: "2021-09-18", because: :versioned_formula
```

Comment:
I'd prefer we leave this out since this is going to keep getting bug fixes. I don't want users getting used to ignoring deprecation notices.